doc/version_history.rst

```diff
@@ -18,6 +18,8 @@ v2.0.0
 * Reorganize constants and event information.
 * Add support for sending start, disable, enable and standby commands.
 * Fix typo in the "ATMCS_nasmyth_m3_mountMotorEncoders" telemetry topic name.
+* Add support for summary state and detailed state events and for FAULT state.
```

Comment:
state events and for FAULT state. --> state events for FAULT state.

Or I misunderstood?

Reply:
I added support for two things:
- the events for summary state and detailed state
- the FAULT state